src/Microsoft.ML.Data/Transforms/ColumnCopying.cs

```diff
@@ -67,6 +81,10 @@ public override SchemaShape GetOutputSchema(SchemaShape inputSchema)
         }
     }
 
+    /// <summary>
+    /// Transformer that creates a new column, named as specified in the output column name parameters, and
+    /// copies the data from the input column to this new column.
```

Comment:
I think ColumnCopyingEstimator should be the main page and this content should go there.

Here we can just say "The transformer resulted from fitting <cref=ColumnCopyingEstimator>" or something short that links back to the estimator.

Reply:
I like it!

---
In reply to: [275069923](https://github.com/dotnet/machinelearning/pull/3316#discussion_r275069923) [](ancestors = 275069923)